vmdb/app/models/orchestration_stack_output.rb

```diff
@@ -3,4 +3,8 @@ class OrchestrationStackOutput < ActiveRecord::Base
 
   belongs_to :stack, :class_name => "OrchestrationStack"
   include ReportableMixin
+
+  def name
+    description
```

Comment:
You can use alias_attribute here: `alias_attribute  :name, :description`

@bzwei Do you think the name attribute should be based off of the 'description' or 'key' column?  I assume key will always be populated.  Is it possible the description could be blank for an outputs instance?